lib/dialects/mssql/query-generator.js

```diff
@@ -354,7 +354,7 @@ var QueryGenerator = {
       if (attribute.type.values && !attribute.values) attribute.values = attribute.type.values;
 
       // enums are a special case
-      template = 'VARCHAR(10) NULL' /* + (attribute.allowNull ? 'NULL' : 'NOT NULL') */;
+      template = 'VARCHAR(255) NULL' /* + (attribute.allowNull ? 'NULL' : 'NOT NULL') */;
       template += ' CHECK (' + attribute.field + ' IN(' + Utils._.map(attribute.values, function(value) {
```

Comment:
Could you move this into the datatype? A funtion called something like `checkConstraint` - so this just becomes

`template = attribute.toSql() + attribute.checkConstraint()`